### DISCUSSION TITLE
何が悪いんでしょう？

### DISCUSSION THREAD

**Discussion by Obutusoba** [TartaricAcid/TouhouLittleMaid#405] [Q&A] [answered]
http://page.cfpa.team/TouhouLittleMaid/　　Wikiを見てmodelを作り、tlm_custom_packにも入れましたが、なぜかゲームに反映されません。何が悪いんでしょうか？
[rekyuu_model_pack-1.0.0.zip](https://github.com/TartaricAcid/TouhouLittleMaid/files/10303794/rekyuu_model_pack-1.0.0.zip)

**Comment by TartaricAcid** [comment]
I'm sorry that I only saw this message now. I don't speak Japanese. I will reply in English.

**Comment by TartaricAcid** [comment, marked as answer]
There was a problem with the json description on the wiki before, I have corrected it

But it is still recommended that you use the tlm-utils plugin to create models
<http://page.cfpa.team/TLM-Utils-Plugins/>
